bindings/python/capstone/__init__.py

```diff
@@ -1221,6 +1222,32 @@ def disasm(self, code, offset, count=0):
             return
             yield
 
+    # This function matches the cs_disasm_iter implementation which
```

Comment:
what's the difference between disasm and diasm (from the PR title)? Or just a typo

Reply:
typo fixed in title.